cd/publish_images.sh

```diff
@@ -55,6 +56,7 @@ pushTags gchq/hdfs "${HADOOP_VERSION}" "${APP_VERSION}"
 pushTags gchq/accumulo "${ACCUMULO_VERSION}" "${APP_VERSION}"
 pushTags gchq/gaffer "${GAFFER_VERSION}" "${APP_VERSION}"
 pushTags gchq/gaffer-rest "${GAFFER_VERSION}" "${APP_VERSION}"
+pushTags gchq/gaffer-ui "${GAFFER_TOOLS_VERSION}" "${APP_VERSION}"
 pushTags gchq/gaffer-road-traffic-loader "${GAFFER_VERSION}" "${APP_VERSION}"
 pushTags gchq/gaffer-operation-runner "${GAFFER_VERSION}" "${APP_VERSION}"
```

Comment:
`gaffer-operation-runner` moved to the gaffer-experimental repository, should we declare a version for that independent of the GAFFER_VERSION
```suggestion
pushTags gchq/gaffer-operation-runner "${GAFFER_EXPERIMENTAL_VERSION}" "${APP_VERSION}"
```